webapp/src/store/Store.spec.ts

```diff
@@ -1,154 +1,163 @@
 import { describe, expect, it } from '@jest/globals';
 
-import Store from './Store';
+import { ProjectStore } from './Store';
 import { LanguageNotFound, MessageNotFound } from '@/errors';
 
-describe('Store', () => {
-  it('returns empty object when empty', async () => {
-    const store = new Store({
-      getTranslations: async () => ({
-        sv: {},
-      }),
-    });
-
-    const translations = await store.getTranslations('sv');
-    expect(translations).toEqual({});
+describe('Store.ts', () => {
```

Comment:
I think these should be separated in two separate files, one class test suite per file.

Reply:
will do

Reply:
done